ui/main/src/app/business/templateGateway/userCardTemplateGateway.ts

```
@@ -0,0 +1,239 @@
+    private static _expirationDate: number;
+    private static _initialKeepChildCards: boolean;
+    private static _initialSeverity: string;
+    private static _lttd: number;
+    private static _processId: string;
+    private static _selectedEntityRecipients: string[];
+    private static _selectedEntityForInformationRecipients: string[];
+    private static _state: string;
+    private static _startDate: number;
+    private static _userEntityChildCard: any;
+
+    private static _functionToSetDropdownEntityRecipientList: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)